rails_event_store/lib/rails_event_store/active_job_dispatcher.rb

```diff
@@ -0,0 +1,64 @@
+require 'active_job'
+
+module RailsEventStore
+  module AsyncProxyStrategy
+    class AfterCommit
+      def call(klass, event)
+        if ActiveRecord::Base.connection.transaction_open?
```

Comment:
That woudln't work with different adapter, right?

Reply:
yep - but this strategy is valid only if you are using ActiveRecord & RESAR event repository